Client/Frontend/Login/LoginInfoViewController.swift

```diff
@@ -0,0 +1,380 @@
+                        
+                        cell.do {
+                            $0.delegate = self
+                            $0.highlightedLabel.text = Strings.Login.loginInfoDetailsWebsiteFieldTitle
+                            $0.descriptionTextField.text = loginEntry.hostname
+                            $0.isEditingFieldData = false
+                            $0.tag = InfoItem.websiteItem.rawValue
+                        }
+                        
+                        websiteField = cell.descriptionTextField
+                        websiteField?.accessibilityIdentifier = "websiteField"
+                        
+                        cell.contentView.alpha = isEditingFieldData ? 0.5 : 1.0
+                        
+                        return cell
+                    case InfoItem.usernameItem.rawValue:
+                        let cell = tableView.dequeueReusableCell(for: indexPath) as LoginInfoTableViewCell
+                      
+                        cell.do {
+                            $0.delegate = self
+                            $0.highlightedLabel.text = Strings.Login.loginInfoDetailsUsernameFieldTitle
+                            $0.descriptionTextField.text = loginEntry.username
+                            $0.descriptionTextField.keyboardType = .emailAddress
+                            $0.descriptionTextField.returnKeyType = .next
+                            $0.isEditingFieldData = isEditingFieldData
+                            $0.tag = InfoItem.usernameItem.rawValue
+                        }
+                        
+                        usernameField = cell.descriptionTextField
+                        usernameField?.accessibilityIdentifier = "usernameField"
+                        
+                        return cell
+                    case InfoItem.passwordItem.rawValue:
+                        let cell = tableView.dequeueReusableCell(for: indexPath) as LoginInfoTableViewCell
+                      
+                        cell.do {
+                            $0.delegate = self
+                            $0.highlightedLabel.text = Strings.Login.loginInfoDetailsPasswordFieldTitle
+                            $0.descriptionTextField.text = loginEntry.password
+                            $0.descriptionTextField.returnKeyType = .done
+                            $0.displayDescriptionAsPassword = true
+                            $0.isEditingFieldData = isEditingFieldData
+                            $0.tag = InfoItem.passwordItem.rawValue
+                        }
+                        
+                        passwordField = cell.descriptionTextField
+                        passwordField?.accessibilityIdentifier = "passwordField"
+                        
+                        return cell
+                    default:
+                        fatalError("No cell available for index path: \(indexPath)")
```

Comment:
assertion failure and return an empty cell

Reply:
Fix at https://github.com/brave/brave-ios/pull/4124/commits/ded5065d24f539a7dee6bb8bcb40b93a2b70fe7b